src/urllib3/contrib/ntlmpool.py

```diff
@@ -5,13 +5,25 @@
 """
 from __future__ import absolute_import
 
+import warnings
 from logging import getLogger
 
 from ntlm import ntlm
 
 from .. import HTTPSConnectionPool
 from ..packages.six.moves.http_client import HTTPSConnection
 
+
+warnings.warn(
+    "The 'urllib3.contrib.nltmpool' module is deprecated and will be removed "
```

Comment:
- "...will be removed in urllib3 v2.0..."
- Remove the mention of the third-party library. We haven't made a decision here yet.
- "If you are a user of this module please comment in the mentioned issue."